signalfx-java/src/test/java/com/signalfx/metrics/connection/HttpDataPointProtobufReceiverConnectionTest.java

```diff
@@ -103,4 +124,76 @@ public boolean isFailed() {
             return false;
         }
     }
+
+    private class BackfillHandler extends AbstractHandler {
+        @Override
+        public void handle(String target, Request baseRequest, HttpServletRequest request,
+                           HttpServletResponse response) throws IOException, ServletException {
+            if(!request.getMethod().equals("POST")) {
+                error("Incorrect HTTP method for backfill", response, baseRequest);
+                return;
+            }
+            if(!request.getRequestURL().toString().endsWith("/v1/backfill")) {
+                error("Incorrect URL for backfill", response, baseRequest);
+            }
+
+            List<NameValuePair> params = URLEncodedUtils.parse(baseRequest.getQueryString(), Charset.forName("UTF-8"));
```

Comment:
Prefer `StandardCharsets.UTF_8`

Reply:
Thanks! IntelliJ complained at me that this was a JDK7 feature and I misread the warning as _after_ JDK7 and was afraid this would break compat. Fixed!